amr-wind/diffusion/incflo_diffusion.cpp

```diff
@@ -133,16 +130,6 @@ void wall_model_bc(
     amrex::Orientation yhi(amrex::Direction::y, amrex::Orientation::high);
     amrex::Orientation zhi(amrex::Direction::z, amrex::Orientation::high);
 
-    // copies cell center to face
-    Real c0 = 1.0;
-    Real c1 = 0.0;
-
-    // linear extrapolate onto face
-    if (extrapolate) {
-        c0 = 1.5;
-        c1 = -0.5;
-    }
-
```

Comment:
Don't we want to keep the "copy cell center to face" case ? This doesn't run the risk of negative viscosity. But I am happy to remove it all together if you don't think we are going to need it in the future

Reply:
that's what I did, the PR only does the copy from cell center to face. Unless I made a typo...